### PR TITLE
Support for es6 re-export syntax

### PR DESCRIPTION
**Pull request by olalonde**
e.g.:

```
export { a } from './a'k
export a from './a'
export * as a from './a'
```

### CODE REVIEW

File: test/es6.js [hunk 1]
```diff
@@ -50,4 +50,9 @@ describe('module format (ES6)', function () {
       format: 'es6'
     }).obj().should.eql({ 'async': [ 'absolute/b' ] });
   });
+  it('should should support export x from "./file"', function() {
```
Comment (mrjoelkemp) [inline test/es6.js]: consider removing the extra `should`

Comment (mrjoelkemp) [inline test/es6.js]: Also consider placing a newline before this for even spacing about the `it`s

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "commander": "^2.8.1",
     "commondir": "^1.0.1",
     "detective": "^4.2.0",
-    "detective-es6": "1.1.4",
+    "detective-es6": "^1.1.5",
```
Comment (mrjoelkemp) [inline package.json]: Not entirely sure if the original specific version peg was on purpose.

**Comment by mrjoelkemp** [comment]
Nice work here @olalonde! Thanks for contributing!

**Comment by olalonde** [comment]
Thanks. fixed your comments. I had copied the test above which also read "should should" so fixed both.

**Comment by mrjoelkemp** [comment]
:+1:

**Comment by pahen** [comment]
I'm on vacation now but will take a look at this as soon I'm back :)

lördag 25 juni 2016 skrev Joel Kemp <notifications@github.com>:

> 👍
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pahen/madge/pull/91#issuecomment-228537175>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAVmYKwuSj04-l6j0vhlz0kWFKsnkQYYks5qPRrigaJpZM4I-RFq>
> .
>


-- 
/Patrik

**Comment by pahen** [comment]
Nice work! Very appreciated :-)

**Pull request merged**